src/js/burials/components/GetFormHelp.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function GetFormHelp() {
+  return (
+    <div>
+      <p className="talk">For benefit related questions please call:</p>
+      <p className="phone-number">
+        <a href="tel:+1-800-827-100">800-827-100</a><br/>
```

Comment:
Thanks @jbalboni, I found these on the burials confirmation page and will include this information here, should they also be on the pensions confirmation page as well?